plugins/src/main/java/com/google/firebase/gradle/plugins/KotlinExtensions.kt

```diff
@@ -26,16 +27,14 @@ fun String.remove(regex: Regex) = replace(regex, "")
 fun String.remove(str: String) = replace(str, "")
 
 /**
- * Returns a sequence containing all elements.
- *
- * The operation is _terminal_.
+ * Joins a variable amount of [strings][Any.toString] to a single [String] split by newlines (`\n`).
  *
- * Syntax sugar for:
- * ```
- * take(count())
+ * For example:
+ * ```kotlin
+ * println(multiLine("Hello", "World", "!")) // "Hello\nWorld\n!"
  * ```
  */
-public fun <T> Sequence<T>.takeAll(): Sequence<T> = take(count())
+fun multiLine(vararg strings: Any?) = strings.joinToString("\n")
```

Comment:
why is this preferable to call `joinToString` directly?

Reply:
I think it looks cleaner, and conveys the purpose better; especially since we do it a lot. Mainly when working with long strings wdyt?
```kt
throw RuntimeException(multiLine(
  "The thing you did was a thing that you really shouldn't do",
  "Like seriously, why did you even try to do that thing",
  "Seek help, and if you need assistance in finding help:",
  "https://gethelp.com"
))
```

```kt
throw RuntimeException(listOf(
  "The thing you did was a thing that you really shouldn't do",
  "Like seriously, why did you even try to do that thing",
  "Seek help, and if you need assistance in finding help:",
  "https://gethelp.com"
).joinToString("\n"))
```